src/app/donation-thanks/donation-thanks.component.html

```diff
@@ -1,211 +1,231 @@
 <main class="b-container">
-  <div *ngIf="complete" id="banner-thank-you"></div>
-
-  <mat-spinner class="complete-spinner" *ngIf="!complete && !noAccess && !timedOut" color="primary"
-               diameter="30" aria-label="Loading donation details"></mat-spinner>
-
-  <div *ngIf="isDataLoaded">
-    <p *ngIf="noAccess" class="error">
-      We can only show your receipt in the tab where you donated, in order to keep your information secure. Don't worry
-      though, we'll email you a receipt within a few minutes confirming if your donation was processed.
-    </p>
-
-    <p *ngIf="timedOut" class="error" aria-live="assertive">
-      Your donation was not processed immediately &ndash; it may be that there is a short delay because we're very
-      busy. Please refresh this page to see the latest status. We'll also email your receipt within a few minutes when
-      your donation is processed.
-    </p>
-
-    <div aria-live="polite">
-      <div *ngIf="complete">
-        <div class="screenreader-only">
-          Thank you! Your generous donation has been made.
-        </div>
+  @if (complete) {
+    <div id="banner-thank-you"></div>
+  }
+
+  @if (!complete && !noAccess && !timedOut) {
+    <mat-spinner class="complete-spinner" color="primary"
+    diameter="30" aria-label="Loading donation details"></mat-spinner>
+  }
+
+  @if (isDataLoaded) {
+    <div>
```

Comment:
this div is probably redundant, would be nice if the script could have removed it and similar, but I don't think it's a big issue.